lib/app/screens/about.dart

```diff
@@ -66,14 +66,50 @@ class VarunExtraInfo extends StatelessWidget {
   }
 }
 
+class MattExtraInfo extends StatelessWidget {
+const MattExtraInfo({super.key});
+
```

Comment:
I'd prefer if we formatted code, you should be able to set an autoformatter.

But I'll do a formatting pass later, so no worries.